<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogmat</class>
 <widget class="QWidget" name="dialogmat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ABCD Matrix</string>
  </property>
  <widget class="QLabel" name="B">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="CustomButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z2real">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z3real">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> + j</string>
   </property>
  </widget>
  <widget class="QLabel" name="C">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z3img">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z1real">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z1img">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z2 =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> + j</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="PiButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PI circuit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Topology:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z2img">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="D">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="A">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> + j</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z1 =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z3 =</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="TButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>T circuit</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>L Circuit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>120</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>1st Element</string>
   </property>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sh</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Series</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shunt</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
